### PR TITLE
Turn off tracking unless HTTPBIN_TRACKING is set.

### LINKED ISSUE #245: Tracking scripts in httpbin package on PyPI
It seems the package released to PyPI contains tracking beacons for Google Analytics and perfectaudience.com

I can see that being useful for httpbin.org (even though I'd prefer another service than Google Analytics, but whatever) - but I really think the version released to PyPI (which could get used e.g. for running it via a CI for some kind of tests) should come without those.

### PR DESCRIPTION
**Pull request by The-Compiler**
Fixes #245.

### CODE REVIEW

File: httpbin/core.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def set_cors_headers(response):
 def view_landing_page():
     """Generates Landing Page."""
 
-    return render_template('index.html')
+    return render_template('index.html', env=os.environ)
```
Comment (johnsheehan) [inline httpbin/core.py]: Instead of passing the whole shebang, how about just a `tracking_enabled` option? Will keep the template cleaner too

**Comment by The-Compiler** [comment]
I moved the environment variable handling out of the template as requested - this now turns tracking on if `HTTPBIN_TRACKING` is set to anything in the environment, but I think that makes more sense than pretending that's a boolean.

**Comment by The-Compiler** [comment]
Any update on this, @johnsheehan @kevin1024? :smile:

**Comment by sigmavirus24** [comment]
Thanks @The-Compiler!

I'd release this but I don't have access to do so.

**Comment by kevin1024** [comment]
OK, new version is up on pypi, thanks @The-Compiler 

@sigmavirus24 I added you to pypi so you can do releases in the future.

**Pull request merged**